### PR TITLE
Color of the Apply button and hover changes, Apply link

### PR DESCRIPTION
**Pull request by christinatong01**
Fixes: #117 

Color and hover properties of button are same as other buttons such as ACM Mailing List on footer, etc

Before hover:
Screen Shot 2021-01-16 at 2 56 01 PM

On hover:
Screen Shot 2021-01-16 at 2 56 12 PM

### CODE REVIEW

**Comment by christinatong01** [comment]
I will fix the check. It didn't pop up when I compiled it in terminal.

File: src/components/MenuBar/ButtonBar.js [hunk 1]
```diff
@@ -35,8 +35,16 @@ function ButtonBar({ isMobile }) {
...
 
 	const BorderLink = ({ ...props }) =>
-		<Button component={Link} role='link' fullWidth={isMobile}
-			className={classNames(classes.btn, classes.borderBtn)} variant='outlined' {...props} />;
+		<Button
+			component={Link}
+			role='link'
+			fullWidth={isMobile}
+			className={classNames(classes.btn)}
+			style = {{ margin: 10 }}
```
Comment (TimothyGu) [inline src/components/MenuBar/ButtonBar.js:43]:
```suggestion
			style={{ margin: 10 }}
```

File: src/components/MenuBar/ButtonBar.js [hunk 2]
```diff
@@ -35,8 +35,16 @@ function ButtonBar({ isMobile }) {
...
+			variant='contained'
+			{...props}
+			color = "secondary"
```
Comment (TimothyGu) [inline src/components/MenuBar/ButtonBar.js:46]:
```suggestion
			color='secondary'
```

File: src/components/MenuBar/ButtonBar.js [hunk 3]
```diff
@@ -63,7 +71,7 @@ function ButtonBar({ isMobile }) {
 				<PoppinLink to={link.to} key={`nav-${index}`}>
 					{link.name}
 				</PoppinLink>)}
-			<BorderLink disabled to='/'>
+			<BorderLink href={'https://forms.gle/7uokDycPQfU9B5oj8'} target='_blank'>
```
Comment (TimothyGu) [inline src/components/MenuBar/ButtonBar.js:74]:
```suggestion
			<BorderLink href='https://forms.gle/7uokDycPQfU9B5oj8' target='_blank'>
```

File: src/components/MenuBar/ButtonBar.js [hunk 4]
```diff
@@ -35,8 +35,16 @@ function ButtonBar({ isMobile }) {
 		<Button component={Link} role='link' fullWidth={isMobile} className={classes.btn} {...props} />;
 
 	const BorderLink = ({ ...props }) =>
-		<Button component={Link} role='link' fullWidth={isMobile}
-			className={classNames(classes.btn, classes.borderBtn)} variant='outlined' {...props} />;
+		<Button
+			component={Link}
+			role='link'
+			fullWidth={isMobile}
+			className={classNames(classes.btn)}
```
Comment (TimothyGu) [inline src/components/MenuBar/ButtonBar.js:42]:
```suggestion
			className={classes.btn}
```

File: src/components/MenuBar/ButtonBar.js [hunk 5]
```diff
@@ -35,8 +35,16 @@ function ButtonBar({ isMobile }) {
 		<Button component={Link} role='link' fullWidth={isMobile} className={classes.btn} {...props} />;
 
 	const BorderLink = ({ ...props }) =>
-		<Button component={Link} role='link' fullWidth={isMobile}
-			className={classNames(classes.btn, classes.borderBtn)} variant='outlined' {...props} />;
+		<Button
+			component={Link}
+			role='link'
+			fullWidth={isMobile}
```
Comment (TimothyGu) [inline src/components/MenuBar/ButtonBar.js:41]:
```suggestion
```

**Pull request merged**